src/Illuminate/Support/Generator.php

```diff
@@ -0,0 +1,383 @@
+<?php
+
+namespace Illuminate\Support;
+
+use Closure;
+use Illuminate\Support\Traits\Macroable;
+use Ramsey\Uuid\Codec\TimestampFirstCombCodec;
+use Ramsey\Uuid\Generator\CombGenerator;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidFactory;
+use Symfony\Component\Uid\Ulid;
+
+class Generator
```

Comment:
Maybe another name for example `StringGenerator`

Reply:
Not bad, let's see what the others say 👍🏻

Reply:
What do you think of `\Illuminate\Support\Str\Generator`?

Reply:
Uhmm that could be a solution but then the other classes also needs to be added to the folder `Str`

Reply:
Yep, that would be the idea then 👍🏻